neovim/api/common.py

```diff
@@ -21,6 +23,7 @@ def __init__(self, session, code_data):
         """
         self._session = session
         self.code_data = code_data
+        self.handle = unpackb(code_data[1])
```

Comment:
It won't work as windows has `number` different from `handle`. It would require a constructor for `Buffer` and I think keeping the property is simpler.